partcad/src/partcad/shape.py

```
@@ -259,13 +267,9 @@
         response = base64.b64decode(response_serialized)
         result = pickle.loads(response)
         if not result["success"]:
-            pc_logging.error(
-                "RenderSVG failed: %s: %s" % (self.name, result["exception"])
-            )
+            pc_logging.error("RenderSVG failed: %s:%s: %s" % (self.project_name, self.name, result["exception"]))
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)